src/gauges/bartio/defaultGaugeList.json

```diff
@@ -268,6 +277,13 @@
       "image": "zeru.svg",
       "name": "Zeru",
       "url": "https://zeru.finance"
+    },
+    {
+      "description": "Participate in Terplayer's testnet event to earn tTERP, and stake your tTERP toearn BGT and more!",
+      "id": "tTERP",
+      "image": "terp.svg",
+      "name": "tTERP Staking",
```

Comment:
```suggestion
      "id": "tterp",
      "image": "terp.svg",
      "name": "tTERP",
```

Reply:
does the first t stand for testing?  If so I would go with "terplayer" as the protocol id but you can keep the extra t for the name or for tokens

Reply:
Yes, the first "t" stands for testing.

Reply:
The name matches the underlying token. 👍 

https://bartio.beratrail.io/token/0xb18CEC4160E5310e79AEf0d06a8df223A35599cD

We'll likely remove the name from this repo in the future and just use the underlying token.